natlas-server/app/api/prepare_work.py

```diff
@@ -19,7 +18,7 @@ def get_target_tags(target):
 def get_unique_scan_id():
 	scan_id = ''
 	while scan_id == '':
-		rand = ''.join(random.choice(string.ascii_lowercase + string.digits) for _ in range(16))
+		rand = secrets.token_hex(16)
```

Comment:
This change decreases the value space of scan ids from 36^16 to 16^16.

Reply:
So `token_hex(16)` generates 16 random bytes and then converts it into hexadecimal. the value of `rand` at the end of this call is a 32 character hexadecimal string. I _think_ this makes the keyspace 256^16 at the expense of the id strings being twice as long.